awx/conf/views.py

```diff
@@ -135,7 +140,7 @@ def perform_destroy(self, instance):
             setting.delete()
             settings_change_list.append(setting.key)
         if settings_change_list:
-            connection.on_commit(lambda: handle_setting_changes.delay(settings_change_list))
+            connection.on_commit(lambda: clear_setting_cache.delay(settings_change_list))
```

Comment:
I think you should still broadcast the rsyslog_configer message here too, because a DELETE removes settings, and can functionally turn external logging off.